flagsmith/flagsmith.py

```diff
@@ -39,18 +47,20 @@ class Flagsmith:
 
     def __init__(
         self,
-        environment_key: str = None,
-        api_url: str = None,
-        custom_headers: typing.Dict[str, typing.Any] = None,
-        request_timeout_seconds: int = None,
+        default_flag_handler: typing.Optional[
+            typing.Callable[[str], DefaultFlag]
+        ] = None,
```

Comment:
Why did you reorder this (and other) argument(s) here? I'm sure people call this with kwargs only, but I don't really see an advantage of reordering them without necessity.

Reply:
Good catch, I was experimenting with whether this really needs a default value or not, and forgot to re-arrange the args back. I will fix this. Thanks!